coordinator/src/logger.rs

```diff
@@ -25,8 +25,7 @@ pub fn init_tracing(level: LevelFilter, json_format: bool, tokio_console: bool)
         .add_directive("rustls=warn".parse()?)
         .add_directive("sled=warn".parse()?)
         .add_directive("bdk=warn".parse()?) // bdk is quite spamy on debug
-        .add_directive("lightning::ln::peer_handler=debug".parse()?)
-        .add_directive("lightning::chain=info".parse()?)
+        .add_directive("lightning=trace".parse()?)
```

Comment:
🤔  I guess that is only temporary, as such I would have opted to make use of the RUST_LOG env variable instead of changing the code.

Simply changing the config in our cloud_conf might have been sufficient?

Reply:
I'm not sure if it should be _that_ temporary. We're still in a phase during which we are discovering bugs often. If we run into an elusive bug and don't have enough context we will be kicking ourselves.